docs/LightningModule/RequiredTrainerInterface.md

```diff
@@ -97,7 +97,7 @@ The LightningModule interface is on the right. Each method corresponds to a part
 ### training_step
 
 ``` {.python}
-def training_step(self, data_batch, batch_nb)
+def training_step(self, batch, batch_nb)
```

Comment:
in this case, I would prefer to stay with `data_batch` of `batch_data`

Reply:
let's keep these consistent. i think we agreed on batch for x_step